SWLOR.Game.Server/Service/AI.cs

```diff
@@ -326,6 +328,22 @@ private static void LoadAggroEffect()
             ApplyEffectToObject(DurationType.Permanent, effect, OBJECT_SELF);
         }
 
+        private static void DoVFX()
+        {
+            // Allow builders to put permanent effects on creatures - e.g. to make them statues, or make them glow.
+            // Index of standard VFX effects here: https://nwnlexicon.com/index.php?title=Vfx_dur
+            int VFX = GetLocalInt(OBJECT_SELF, "VFX");
```

Comment:
We have a script to do something similar to this, but for placeables. It's in the PlaceableScripts.cs feature. On that one we've got the variable named PERMANENT_VFX_ID

This is probably nit picky but if it's not too much trouble can we use the same variable name here for consistency?

Reply:
sure thing - let me do that now.